src/pages/Missions/Missions.js

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMissionAsync, joiningMission, leavingMission } from '../../redux/missionSlice';
+import './Missions.css';
+
+const Missions = () => {
+  const dispatch = useDispatch();
+  const missions = useSelector((state) => state.missions.missions);
+  const status = useSelector((state) => state.missions.status);
+  const error = useSelector((state) => state.missions.error);
+
+  useEffect(() => {
+    if (missions.length === 0) {
+      dispatch(fetchMissionAsync());
+    }
+  }, [dispatch]);
+
+  if (status === 'loading') {
+    return <div>Loading...</div>;
+  }
+
+  if (status === 'failed') {
+    return (
+      <div>
+        Error:
+        {error}
+      </div>
+    );
+  }
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>Mission</th>
+          <th>Description</th>
+          <th>Status</th>
+          <th> </th>
+        </tr>
+      </thead>
+      <tbody className="border border-gray-300">
+        {missions.map((mission) => (
+          <tr key={mission.mission_id}>
+            <td>{mission.mission_name}</td>
+            <td>{mission.description}</td>
+            <td>
+              <p
+                style={{
+                  backgroundColor: mission.reserved ? '#419bf9' : '#6d757d',
+                }}
+              >
+                {mission.reserved ? 'ACTIVE MEMBER' : 'NOT A MEMBER'}
+              </p>
+            </td>
+            <td>
+              {mission.reserved && (
+              <button
+                type="button"
+                style={{
+                  color: mission.reserved ? 'red' : '',
+                  border: mission.reserved ? '1px solid red' : '',
+                }}
+                onClick={() => dispatch(leavingMission(mission.mission_id))}
+              >
+                Leave Mission
+              </button>
+              )}
+              {!mission.reserved && (
+              <button
+                type="button"
+                style={{
+                  color: mission.reserved ? 'red' : '',
+                  border: mission.reserved ? '1px solid red' : '',
+                }}
+                onClick={() => dispatch(joiningMission(mission.mission_id))}
+              >
+                Join Mission
+              </button>
+              )}
+            </td>
+          </tr>
+        ))}
+      </tbody>
+    </table>
```

Comment:
👍 Your mission pages are functioning correctly, but there's a minor layout issue that needs attention to align with the design:

- [ ] If you examine the design closely, you'll notice that the left and right sides of your table should horizontally align with the navigation bar. I recommend updating your code to achieve this alignment and ensure that the layout matches the design as closely as possible.

| Your | Design |
| --- | --- |
|  ![Frame 1 (2)](https://github.com/pushkar1995/space-travelers-hub/assets/55840999/b34d3e6e-c453-4223-8a1e-f8d7bfbf5f62)| ![Frame 2](https://github.com/pushkar1995/space-travelers-hub/assets/55840999/9becc25c-7240-492d-8898-5f21fa0bf51d) |



By making this adjustment, you'll enhance the visual consistency of your mission pages and bring them in line with the design. Keep up the great work! 👍🎨